.github/ISSUE_TEMPLATE/crash.md

```diff
@@ -5,29 +5,37 @@ labels: "type-crash"
 ---
 
 <!--
-  Use this template for hard crashes of the interpreter, segmentation faults, failed C-level assertions, and similar.
-  Do not submit this form if you encounter an exception being unexpectedly raised from a Python function.
-  Most of the time, these should be filed as bugs, rather than crashes.
+  This form is for hard crashes of the Python interpreter, segmentation faults,
+  failed C-level assertions, and similar.
+  Exceptions unexpectedly raised from stdlib Python functions
+  count as bugs rather than crashes.
 
-  The CPython interpreter is itself written in a different programming language, C.
-  For CPython, a "crash" is when Python itself fails, leading to a traceback in the C stack.
+  The CPython interpreter is written in a different programming language, C.
+  A " CPython crash" is when Python itself fails, leading to a traceback in the C stack.
 -->
 
 # Crash report
 
-Tell us what happened, ideally including a minimal, reproducible example (https://stackoverflow.com/help/minimal-reproducible-example).
+<!--
+  Tell us what happened. Ideally, include a minimal, reproducible example.
+  (https://stackoverflow.com/help/minimal-reproducible-example)
+-->
+
+
 
 # Error messages
 
-Enter any relevant error message caused by the crash, including a core dump if there is one.
+<!-- Enter any error messages caused by the crash, including a core dump if there is one -->
```

Comment:
This could mention the `<details>/<summary>` trick.

Reply:
Hmmmm. Could do. _But_, one of the main aims of this PR is to try to make these templates more concise, so that people actually read the whole thing. Not sure if mentioning the `<details>` trick here is worth the cost to brevity — WDYT?

Reply:
I was thinking something like:
```html
<!--
<details>
<summary>Core dump</summary>

If you have a core dump paste it here and uncomment this block.
</details>
-->
```

I'm not sure the problem this is trying to solve (avoiding having a huge log pasted in the issue) is common enough to make the addition of this section worthy.  If you think it's not, feel free to ignore my suggestions.  Core devs (and maybe triagers too) can also edit the comments to add those tags after the fact.

Reply:
> I'm not sure the problem this is trying to solve (avoiding having a huge log pasted in the issue) is common enough to make the addition of this section worthy.

I feel like I see huge logs pasted in issues relatively rarely. But I see badly formatted code blocks quite a lot -- so I've added notes in the templates about using triple backticks to surround code examples.